-meeting-template.md

```diff
@@ -8,31 +8,43 @@ filename: -meeting-template.md
 [:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
 
 Time: ...  
-Location: https://meet.jit.si/offline  
+Location: https://meet.jit.si/hyphacoop  
 Attending: ...
 
 [TOC]
 
 ## Agenda
 
+- Previous Action Items
```

Comment:
Would a meeting index like this be helpful to corral these? https://beta.hackfoldr.org/hyphacoop-meetings/

Pros:
- accessible even if not a contributor to hackmd pad
- can disregard figuring out which note-taking tool has recent edition when trying to find notes
- can link a single "meeting index" that is possible to be consistent and reliably complete
- can all pass through same path when looking for notes, and put it on a shared "critical path" -- No more sharing links to notes directly, but can just say, go to meeting index.
- can put meeting index link in gcal events, instead of custom links.
- can more easily see when agendas have been created for future meetings
- can point to open PRs when notes are awaiting merge.

Cons:
- pros only work if it's in our process to ensure updateness.
- another tool

Alternatively, maybe a task tracker would make this simpler too, for example:

![Screen Shot 2019-06-10 at 4 45 56 PM](https://user-images.githubusercontent.com/305339/59225637-42e6e580-8b9f-11e9-8a65-5a2153449901.png)